src/components/ResultCard/ResultCard.tsx

```diff
@@ -49,24 +89,20 @@ export default function ResultCard(props: ResultCardProps) {
     </>
```

Comment:
Should we pass price to the ResultCtaButton?

Reply:
Good catch. I had it then I accidentally removed it when I was reorganizing code here [6e20c11](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/56/commits/6e20c117e4144c3d33185457edb5ff5b4d5015c8)